.travis.yml

```diff
@@ -20,49 +20,33 @@ addons:
       - net-tools
       - libpcre3-dev
       - build-essential
+  hosts:
+    - grpcs_1.test
+    - grpcs_2.test
 
 env:
   global:
-    - OPENRESTY=1.13.6.2
-    - KONG_BRANCH=master
-    - KONG_NGINX_MODULE_BRANCH=master
-    - OPENSSL=1.1.1c
-    - LUAROCKS=3.1.2
-    - DOWNLOAD_ROOT=$HOME/download-root
-    - DOWNLOAD_CACHE=$HOME/download-cache
+    - KONG=chore/ci-setup
+    - OPENRESTY_PATCHES_BRANCH=$TRAVIS_BRANCH
     - INSTALL_CACHE=$HOME/install-cache
+    - DOWNLOAD_ROOT=$HOME/download-root
     - KONG_TEST_PG_DATABASE=travis
     - KONG_TEST_PG_USER=postgres
     - JOBS=2
   matrix:
     - KONG_TEST_DATABASE=postgres TEST_SUITE=integration
     - KONG_TEST_DATABASE=cassandra CASSANDRA=2.2 TEST_SUITE=integration
     - KONG_TEST_DATABASE=cassandra CASSANDRA=3.9 TEST_SUITE=integration
+    - KONG_TEST_DATABASE=off TEST_SUITE=dbless
     - KONG_TEST_DATABASE=postgres TEST_SUITE=plugins
     - KONG_TEST_DATABASE=cassandra CASSANDRA=2.2 TEST_SUITE=plugins
     - KONG_TEST_DATABASE=cassandra CASSANDRA=3.9 TEST_SUITE=plugins
     - TEST_SUITE=pdk
-    - OPENRESTY=1.15.8.1 KONG_BRANCH=next KONG_TEST_DATABASE=postgres TEST_SUITE=integration
-    - OPENRESTY=1.15.8.1 KONG_BRANCH=next KONG_TEST_DATABASE=cassandra CASSANDRA=2.2 TEST_SUITE=integration
-    - OPENRESTY=1.15.8.1 KONG_BRANCH=next KONG_TEST_DATABASE=cassandra CASSANDRA=3.9 TEST_SUITE=integration
-    - OPENRESTY=1.15.8.1 KONG_BRANCH=next KONG_TEST_DATABASE=postgres TEST_SUITE=plugins
-    - OPENRESTY=1.15.8.1 KONG_BRANCH=next KONG_TEST_DATABASE=cassandra CASSANDRA=2.2 TEST_SUITE=plugins
-    - OPENRESTY=1.15.8.1 KONG_BRANCH=next KONG_TEST_DATABASE=cassandra CASSANDRA=3.9 TEST_SUITE=plugins
-    - OPENRESTY=1.15.8.1 KONG_BRANCH=next TEST_SUITE=pdk
 
 install:
-  - source .ci/setup_env.sh
-
-cache:
-  apt: true
-  pip: true
-  directories:
-    - $OPENSSL_DOWNLOAD
-    - $LUAROCKS_DOWNLOAD
-    - $CPAN_DOWNLOAD
-    - $OPENSSL_INSTALL
-    - $LUAROCKS_INSTALL
-    - $HOME/.ccm/repository
+  - make setup-ci
+  - pushd kong && source .ci/setup_env.sh && popd
```

Comment:
We need to run the make task as well as `source` the bash script so the bash script can export the necessary environment variables which isn't possible via Make